help/compute/diff-studio.md

```diff
@@ -35,11 +35,29 @@ To run **Diff Studio**:
 
 ### Loading and saving data
 
-* **To save formulas** to a local file, click the <i class="fas fa-save"></i> **Save** icon on the ribbon, and find the *ivp*-file in Downloads. You can open and edit this file using any text editor.
+* **To save formulas**, click the <i class="fas fa-save"></i> **Save** icon on the ribbon. The context menu opens:
```

Comment:
Probably let's use "model" instead of "formulas" consistently

Reply:
Fixed